source/_patterns/01-atoms/list-group/index.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * List-group
+ */
+import 'bootstrap/js/src/tab';
```

Comment:
Noting here that we will probably build a standalone Tabs component and also require this there. Yay webpack!